solr/solrj/src/test/org/apache/solr/client/solrj/SolrExampleTests.java

```diff
@@ -2065,9 +2074,9 @@ public void testChildDoctransformer() throws IOException, SolrServerException {
         q = new SolrQuery("q", "name:" + name, "indent", "true");
       }
       q.setFilterQueries(parentFilter);
-      q.setFields("id, level_i, [child parentFilter=\"" + parentFilter +
-                  "\" childFilter=\"" + childFilter + 
-                  "\" limit=\"" + maxKidCount + "\"],name");
+      q.setFields("id, level_i, [child " +
```

Comment:
Can you add newlines at the comma of fields instead of in the middle of the transformer?
You could call q.addField multiple times instead of setFields; which would look nicer?